crates/rattler/src/cli/auth.rs

```diff
@@ -24,6 +24,18 @@ pub struct LoginArgs {
     /// The token to use on anaconda.org / quetz authentication
     #[clap(long)]
     conda_token: Option<String>,
+
+    /// The AWS access key ID
+    #[clap(long)]
+    aws_access_key_id: Option<String>,
```

Comment:
maybe `access_key_id` would be more fitting...

Reply:
Changed to `s3_*`.